### PR TITLE
Add tests for Fablicop::CLI

### PR DESCRIPTION
**Pull request by yykamei**
I added tests for `Fablicop::CLI` for its refactoring later.
I hope this satisfies the current behaviors expectedly.

### CODE REVIEW

File: .github/workflows/run-ci.yml [hunk 1]
```diff
@@ -15,10 +15,8 @@ jobs:
...
-    - name: Set up Ruby
-      uses: ruby/setup-ruby@v1
+    - uses: ruby/setup-ruby@v1
       with:
         ruby-version: ${{ matrix.ruby }}
-        bundler-cache: true # runs 'bundle install' and caches installed gems automatically
-    - name: Run rspecs
-      run: bundle exec rspec
+        bundler-cache: true
+    - run: bundle exec rspec
```
Comment (yykamei) [inline .github/workflows/run-ci.yml:22]: I think each `run` is explicit enough, so I just deleted `name` as well as the comment for `bundler-cache`.

**Review by yo-it-engineer — APPROVED** [review]
LGTM! Thanks for adding the spec.

**Comment by yo-it-engineer** [comment]
memo: Fablicop::CLI -> https://github.com/Fablic/fablicop/blob/master/lib/fablicop/cli.rb

**Comment by yykamei** [comment]
Thank you!

**Pull request merged**